frequenz/resampling/_rust_backend.pyi

```diff
@@ -0,0 +1,76 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+__all__ = "Resampler", "ResamplingFunction"
+
+from datetime import datetime, timedelta
+from typing import Optional
+
+class ResamplingFunction:
+    """
+    The ResamplingFunction enum represents the different resampling functions
+    that can be used to resample a time series.
+    """
+
+    Average: ResamplingFunction
```

Comment:
Oh, this is interesting, so this is similar to enum in the sense that the class has attributes that are instances of itself? Does it offer any way to create from a regular `int` like `Enum`?

Do you have any links to docs for pyo3 that describe how python types are built?

Reply:
Unfortunately, it does not have a constructor and, thus, `ResamplingFunction(1)` does not work.

This gives some information about building an Enum: https://pyo3.rs/v0.23.1/class.html#defining-a-new-class

Reply:
* https://github.com/frequenz-floss/frequenz-resampling-rs/issues/4